src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -98,88 +97,78 @@
 struct NaClDescQuotaInterface;
 struct NaClSyscallTableEntry nacl_syscall[NACL_MAX_SYSCALLS];
 
-// Translate Host FD to NaclDesc
-// if functions returns NULL, calling function should return -NACL_ABI_EBADF
-struct NaClDesc *GetDescFromCagetable(struct NaClApp *nap, int fd) {
-
-  if ((fd >= FILE_DESC_MAX) || (fd < 0)) {
-    return NULL;
-  }
-
-  int naclfd = fd_cage_table[nap->cage_id][fd];
-  if (naclfd < 0) {
-    return NULL;
-  }
-  
-  return NaClGetDesc(nap, naclfd);
-}
-
-// extract lindfd from NaClDesc
-int NaClDesc2Lindfd(struct NaClDesc * ndp) {
-  return ((struct NaClDescIoDesc *) ndp)->hd->d;
-}
-
-int32_t NaClSysNotImplementedDecoder(struct NaClAppThread *natp) {
+int32_t NaClSysNotImplementedDecoder(struct NaClAppThread *natp)
+{
   return -NACL_ABI_ENOSYS;
 }
 
-void NaClAddSyscall(int num, int32_t (*fn)(struct NaClAppThread *)) {
-  if (nacl_syscall[num].handler != &NaClSysNotImplementedDecoder) {
+void NaClAddSyscall(int num, int32_t (*fn)(struct NaClAppThread *))
+{
+  if (nacl_syscall[num].handler != &NaClSysNotImplementedDecoder)
+  {
     NaClLog(LOG_FATAL, "Duplicate syscall number %d\n", num);
   }
   nacl_syscall[num].handler = fn;
 }
 
-int32_t NaClSysNull(struct NaClAppThread *natp) {
+int32_t NaClSysNull(struct NaClAppThread *natp)
+{
   UNREFERENCED_PARAMETER(natp);
   return 0;
 }
 
 int32_t NaClSysBrk(struct NaClAppThread *natp,
-                   uintptr_t            new_break) {
-  struct NaClApp        *nap = natp->nap;
-  uintptr_t             break_addr;
-  int32_t               rv = -NACL_ABI_EINVAL;
-  struct NaClVmmapIter  iter;
+                   uintptr_t new_break)
+{
+  struct NaClApp *nap = natp->nap;
+  uintptr_t break_addr;
+  int32_t rv = -NACL_ABI_EINVAL;
+  struct NaClVmmapIter iter;
   struct NaClVmmapEntry *ent;
   struct NaClVmmapEntry *next_ent;
-  uintptr_t             sys_break;
-  uintptr_t             sys_new_break;
-  uintptr_t             usr_last_data_page;
-  uintptr_t             usr_new_last_data_page;
-  uintptr_t             last_internal_data_addr;
-  uintptr_t             last_internal_page;
-  uintptr_t             start_new_region;
-  uintptr_t             region_size;
+  uintptr_t sys_break;
+  uintptr_t sys_new_break;
+  uintptr_t usr_last_data_page;
+  uintptr_t usr_new_last_data_page;
+  uintptr_t last_internal_data_addr;
+  uintptr_t last_internal_page;
+  uintptr_t start_new_region;
+  uintptr_t region_size;
 
   break_addr = nap->break_addr;
 
-  NaClLog(3, "Entered NaClSysBrk(new_break 0x%08"NACL_PRIxPTR")\n",
+  NaClLog(3, "Entered NaClSysBrk(new_break 0x%08" NACL_PRIxPTR ")\n",
           new_break);
 
   sys_new_break = NaClUserToSysAddr(nap, new_break);
-  NaClLog(3, "sys_new_break 0x%08"NACL_PRIxPTR"\n", sys_new_break);
+  NaClLog(3, "sys_new_break 0x%08" NACL_PRIxPTR "\n", sys_new_break);
 
-  if (kNaClBadAddress == sys_new_break) {
+  if (kNaClBadAddress == sys_new_break)
+  {
```

Comment:
Particularly doing this with the curly braces, breaking K&R style, adds a lot of vertical space to the file that the style used elsewhere does not.

Reply:
In general, I can't give this file a good review because there are too many style changes and spacing changes. I suspect that you ran an autoformatter on this, but it really does clutter up the diff to the point of being a lot harder to work on. If you did want to change this many stylistic things, it should have been done in a different PR. Could you extricate these if they are in a single commit?